clusterprovider/kind/kind.go

```diff
@@ -75,7 +64,7 @@ func (k KindClusterProvider) Delete(ctx context.Context,
 	return k.kindProvider.Delete(name, opts.KubeconfigPath)
 }
 
-func (k KindClusterProvider) List() ([]string, error) {
+func (k KindClusterProvider) ListNames(ctx context.Context) ([]string, error) {
```

Comment:
I would rename to ListClusterNames to avoid use of the overloaded Name.

Reply:
Changed to ListClusters & ListClusterNames